style.css

```diff
@@ -107,3 +110,357 @@ body {
 .fab {
   color: #505f79;
 }
+
+/* works section starts here */
+
+.works-section-container {
+  background-color: #e5e5e5;
+  width: 100%;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  margin-top: 114px;
+  display: grid;
+  grid-template-rows: 552px 552px 552px 552px;
+  row-gap: 88px;
+}
+
+.card-container {
+  background-color: #fff;
+  margin: 0 24px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  gap: 12px;
+  border-radius: 16px;
+}
+
+.card-type1 {
+  width: 327px;
+  height: 552px;
+}
+
+.card-type2 {
+  width: 327px;
+  height: 596px;
+}
```

Comment:
- [ ] You shouldn't fix the `height` and `width` of the cards, instead, you can provide a `min-width` and a `max-width` and control the rest using margins and padding, this will let the container control the size of each card, kindly change this accordingly. 😁